Missing field 'network' from com.vmware.vim25.CannotAccessNetwork fault

Hi,

I am getting the following exception while getting the events with a _EventHistoryCollector_:

```
java.rmi.RemoteException: Exception in SoapClient.invoke:; nested exception is: 
    java.lang.NoSuchFieldException: network
    at com.vmware.vim25.ws.XmlGenDom.fromXML(XmlGenDom.java:123)
    at com.vmware.vim25.ws.SoapClient.unMarshall(SoapClient.java:212)
    at com.vmware.vim25.ws.WSClient.invoke(WSClient.java:93)
    at com.vmware.vim25.ws.VimStub.retrieveProperties(VimStub.java:106)
    at com.vmware.vim25.mo.PropertyCollector.retrieveProperties(PropertyCollector.java:98)
    at com.vmware.vim25.mo.ManagedObject.retrieveObjectProperties(ManagedObject.java:146)
    at com.vmware.vim25.mo.ManagedObject.getCurrentProperty(ManagedObject.java:170)
    at com.vmware.vim25.mo.EventHistoryCollector.getLatestPage(EventHistoryCollector.java:56)
...
Caused by: java.lang.NoSuchFieldException: network
    at java.lang.Class.getField(Class.java:1703)
    at com.vmware.vim25.ws.XmlGenDom.fromXml(XmlGenDom.java:233)
    at com.vmware.vim25.ws.XmlGenDom.fromXml(XmlGenDom.java:329)
    at com.vmware.vim25.ws.XmlGenDom.fromXml(XmlGenDom.java:329)
    at com.vmware.vim25.ws.XmlGenDom.fromXml(XmlGenDom.java:322)
    at com.vmware.vim25.ws.XmlGenDom.fromXml(XmlGenDom.java:329)
    at com.vmware.vim25.ws.XmlGenDom.fromXml(XmlGenDom.java:322)
    at com.vmware.vim25.ws.XmlGenDom.fromXML(XmlGenDom.java:204)
    at com.vmware.vim25.ws.XmlGenDom.fromXML(XmlGenDom.java:120)
    ... 12 more
Caused by: java.lang.RuntimeException: java.rmi.RemoteException: Exception in SoapClient.invoke:; nested exception is: 
    java.lang.NoSuchFieldException: network
    at com.vmware.vim25.mo.ManagedObject.retrieveObjectProperties(ManagedObject.java:149)
    at com.vmware.vim25.mo.ManagedObject.getCurrentProperty(ManagedObject.java:170)
    at com.vmware.vim25.mo.EventHistoryCollector.getLatestPage(EventHistoryCollector.java:56)
    ... 4 more
Caused by: java.rmi.RemoteException: Exception in SoapClient.invoke:; nested exception is: 
    java.lang.NoSuchFieldException: network
    at com.vmware.vim25.ws.XmlGenDom.fromXML(XmlGenDom.java:123)
    at com.vmware.vim25.ws.SoapClient.unMarshall(SoapClient.java:212)
    at com.vmware.vim25.ws.WSClient.invoke(WSClient.java:93)
    at com.vmware.vim25.ws.VimStub.retrieveProperties(VimStub.java:106)
    at com.vmware.vim25.mo.PropertyCollector.retrieveProperties(PropertyCollector.java:98)
    at com.vmware.vim25.mo.ManagedObject.retrieveObjectProperties(ManagedObject.java:146)
    ... 7 more
Caused by: java.lang.NoSuchFieldException: network
    at java.lang.Class.getField(Class.java:1703)
    at com.vmware.vim25.ws.XmlGenDom.fromXml(XmlGenDom.java:233)
    at com.vmware.vim25.ws.XmlGenDom.fromXml(XmlGenDom.java:329)
    at com.vmware.vim25.ws.XmlGenDom.fromXml(XmlGenDom.java:329)
    at com.vmware.vim25.ws.XmlGenDom.fromXml(XmlGenDom.java:322)
    at com.vmware.vim25.ws.XmlGenDom.fromXml(XmlGenDom.java:329)
    at com.vmware.vim25.ws.XmlGenDom.fromXml(XmlGenDom.java:322)
    at com.vmware.vim25.ws.XmlGenDom.fromXML(XmlGenDom.java:204)
    at com.vmware.vim25.ws.XmlGenDom.fromXML(XmlGenDom.java:120)
    ... 12 more
```

The problem seems to come from the fact that 'network' property is missing from [CannotAccessNetwork](http://pubs.vmware.com/vsphere-60/index.jsp#com.vmware.wssdk.apiref.doc/vim.fault.CannotAccessNetwork.html) fault. By looking at the property description it looks like this was added in vSphere 6.0, and probably forgotten in yavijava.

It is not obvious how to produce a test class to highlight the problem mostly because I am pretty sure it's circumstantial. Maybe this will help: this exception is generated by my events collector once a virtual machine is cloned. 

Here's my environment:
Java: 1.8.0_66-b17
yavijava:  6.0.03
vSphere: 6.0.0

Any help would be appreciated! Thanks!
Alex

--- a/src/main/java/com/vmware/vim25/CannotAccessNetwork.java
+++ b/src/main/java/com/vmware/vim25/CannotAccessNetwork.java
@@ -29,6 +29,9 @@ POSSIBILITY OF SUCH DAMAGE.
 
 package com.vmware.vim25;
 
+import lombok.Getter;
+import lombok.Setter;
+
 /**
  * @author Steve Jin (http://www.doublecloud.org)
  * @version 5.1
@@ -36,4 +39,5 @@ package com.vmware.vim25;
 
 @SuppressWarnings("all")
 public class CannotAccessNetwork extends CannotAccessVmDevice {
+    @Getter @Setter public ManagedObjectReference network;
 }

